### PR TITLE
ADDED Itinerary class implemented and tested

### PR DESCRIPTION
**Pull request by earandap**
Please check this

### CODE REVIEW

File: t/itinerary.t [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+use strict;
+use warnings;
+use v5.18;
+use Test::More;
+
+use lib '../lib';
+use Itinerary;
+
+my $date  = localtime();
+say $date;
```
Comment (elin3t) [inline t/itinerary.t]: why 'say $date' ??

**Comment by earandap** [comment]
ready!

**Comment by elin3t** [comment]
+1

**Comment by DiegoteGitHub** [comment]
+1

**Pull request merged**